docs/WritingTests.asciidoc

```diff
@@ -281,8 +281,12 @@ Job dependencies are only resolved when using the iso controller to
 create new jobs from job templates. Posting individual jobs manually
 won't work.
 
-Job dependencies are currently only possible between tests that are
-scheduled for the same machine.
+Job dependencies now allow inter-machine dependency that make possible between tests that are scheduled for the different machine.
+To use it simply append machine for each dependent test with colon separated.
+E.g. +START_AFTER_TEST="kde:64bit-1G,dhcp-server:64bit-8G"+
+or
+E.g. +PARALLEL_WITH="web-server:ipmi-fly,dhcp-server:ipmi-bee"+
+Without specifying machine means the dependency tests that are scheduled for the same machine.
```

Comment:
This is not really proper english, but before correcting it: is that really the case? Because you have a test case for best match, but no mentioning here.